src/control/security/auth_sys.go

```diff
@@ -108,17 +122,15 @@ func AuthSysRequestFromCreds(ext UserExt, creds *DomainInfo) (*auth.Credential,
 	// Marshal our AuthSys token into a byte array
 	tokenBytes, err := proto.Marshal(&sys)
 	if err != nil {
-		fmt.Errorf("Unable to marshal AuthSys token (%s)", err)
-		return nil, err
+		return nil, fmt.Errorf("Unable to marshal AuthSys token (%s)", err)
```

Comment:
could use errors.WithMessagef here

Reply:
I think a few of the security files still need to be converted to use the errors package. Is there already a ticket? I can address it in a new patch.

Reply:
Couldn't find an existing ticket, so I created one: https://jira.hpdd.intel.com/browse/DAOS-2560